circle.yml

```diff
@@ -34,13 +37,13 @@ dependencies:
     - wget --no-check-certificate https://github.com/ninja-build/ninja/releases/download/v1.7.2/ninja-linux.zip
     - unzip ninja-linux.zip && sudo cp ninja /usr/local/bin/ && sudo chmod 755 /usr/local/bin/ninja
     # Use scikit-ci-addons to streamline installation of CMake
-    - sudo pip install scikit-ci-addons==0.11.0
+    - pip install scikit-ci-addons==0.11.0
     - ci_addons circle/install_cmake 3.6.2
 
   post:
     - mkdir -p ~/dcmqi-build && cd ~/dcmqi-build && cmake -G Ninja ../dcmqi && ninja
     # Install package required by convert.py
-    - sudo pip install lxml==3.6.4
+    - pip install lxml==3.6.4
```

Comment:
Moving forward, these could be added to a file named `requirements-circleci.txt`